content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Full speed XMPP universe! 04 Feb 2020
+save_as: 2020/04/newsletter-04-february/index.html
+url: 2020/02/newsletter-04-february/
+date: 2020-02-04 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of January 2020, which has seen a lot of activity!
+
+Help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Ingo Jürgensmann has written two interesting articles:
+
+* [XMPP - Prosody & Ejabberd](https://blog.windfluechter.net/content/blog/2020/01/07/1754-xmpp-prosody-ejabberd)
+* [XMPP - Fun with Clients](https://blog.windfluechter.net/content/blog/2020/01/18/1755-xmpp-fun-clients)
+
+David Wong wrote an insightful point by point piece: [A history of end-to-end encryption and the death of PGP](https://www.cryptologie.net/article/487/a-history-of-end-to-end-encryption-and-the-death-of-pgp/)
+
+This Twitter thread started by Thierry Stoehr shows [various use cases of XMPP in Instant Messaging](https://twitter.com/ThierryStoehr/status/1202978443048235009): healthcare, miltary, police and law enforcement, municipal services, intelligence agencies, in-games chat... Please "Like" and "Share", and even add the use cases you have seen in action.
+
+Mike Kuketz started of a series about different messengers including XMPP: [Die verrückte Welt der Messenger – Messenger Teil1](https://www.kuketz-blog.de/die-verrueckte-welt-der-messenger-messenger-teil1/) (German).
+
+![Federation](https://media.kuketz.de/blog/artikel/2020/welt-der-messenger/federation.png)
```

Comment:
We shouldn't link external images on our blog, nobody knows how long they will remain. It's probably better to add a local copy and have a short source note in the description / commit message.

Reply:
Agree: how do we do that?

Reply:
Download a copy of the image to `content/images/blog`, add it to git, reference it as `/images/blog/filename.png`

Reply:
I use the web interface of GitHub: how do you add file to the repo you pointed, in the same PR?

Reply:
I could upload the files in the dirs from GitHub Desktop, but that's one more tool (= one more context switch), I got lost in the middle, could find back my path, but could not change the URLs: what should they be? How can you test and validate both on this GitHub interface *AND* Pelican?

Reply:
I'm not sure it's possible. Maybe just go on with it as-is now, and we'll replace the image+link after the newsletter is published.

Reply:
How?

Reply:
I mean, a change in the code is deployed when merged, but it takes about 30 min... so cannot test immediately, cannot fix immediately.

Reply:
Once again, for this newsletter, I will not host the images on our server.
I'd to investigate how to do it, for this, I'll need help.